test/scenarios/support_archive/install.go

```diff
@@ -120,16 +114,16 @@ func executeSupportArchiveCommand(ctx context.Context, t *testing.T, environment
 }
 
 func collectRequiredFiles(t *testing.T, ctx context.Context, resources *resources.Resources, testDynakube dynatracev1beta1.DynaKube) []string {
-	namespace := testDynakube.Namespace
+	n := testDynakube.Namespace
```

Comment:
```suggestion
	ns := testDynakube.Namespace
```
IMO is more readable name.

Reply:
I think similar rename you did with `context` -> `ctx`.